examples/06-Multiphysics/Maxwell3D_Icepak_2Way_Coupling.py

```diff
@@ -0,0 +1,293 @@
+
+solution_setup = ipk.create_setup()
+solution_setup.props["Convergence Criteria - Max Iterations"] = 50
+solution_setup.props["Flow Regime"] = "Turbulent"
+solution_setup.props["Turbulent Model Eqn"] = "ZeroEquation"
+solution_setup.props["Radiation Model"] = "Discrete Ordinates Model"
+solution_setup.props["Include Flow"] = True
+solution_setup.props["Include Gravity"] = True
+solution_setup.props["Solution Initialization - Z Velocity"] = "0.0005m_per_sec"
+solution_setup.props["Convergence Criteria - Flow"] = 0.0005
+solution_setup.props["Flow Iteration Per Radiation Iteration"] = "5"
+
+
+###############################################################################
+# Add 2-way coupling and solve the project
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Enable mapping temperature distribution back to Maxwell.
+# Default number Maxwell <--> Icepak iterations is 2,
+# but for increased accuracy it can be increased (number_of_iterations).
+
+ipk.assign_2way_coupling()
+ipk.analyze_setup(name=solution_setup.name)
+
+###############################################################################
+# Postprocessing
+# ~~~~~~~~~~~~~~
+# Plot temperature on the object surfaces.
+
+surface_list = []
+for name in ["Coil", "Core"]:
+    surface_list.extend(ipk.modeler.get_object_faces(name))
+
+surf_temperature = ipk.post.create_fieldplot_surface(
+    surface_list,
+    quantityName="SurfTemperature",
+    plot_name="Surface Temperature"
+)
+
+velocity_cutplane = ipk.post.create_fieldplot_cutplane(
+        objlist="Global:XZ",
+        quantityName="Velocity Vectors",
+        plot_name="Velocity Vectors"
+    )
+
+surf_temperature.export_image()
+velocity_cutplane.export_image(orientation="right")
+
+report_temp = ipk.post.create_report(expressions="PointMonitor1.Temperature", primary_sweep_variable="X")
+solution_temp = report_temp.get_solution_data()
+temp = solution_temp.data_magnitude()[0]
+m3d.logger.info("*******Coil temperature =  {:.2f}deg C".format(temp))
+
+###############################################################################
+# Get new resistance from Maxwell
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Temperature of the coil increases, and consequently also coil resistance increases.
+
+report_new = m3d.post.create_report(expressions="Matrix1.R(Winding1,Winding1)")
+solution_new = report_new.get_solution_data()
+resistance_new = solution_new.data_magnitude()[0]
+resistance_increase = (resistance_new - resistance)/resistance * 100
+
+report_loss_new = m3d.post.create_report(expressions="StrandedLossAC")
+solution_loss_new = report_loss_new.get_solution_data()
+em_loss_new = solution_loss_new.data_magnitude()[0]
+
+m3d.logger.info("*******Coil resistance at 150kHz AFTER temperature feedback =  {:.2f}Ohm".format(resistance_new))
+m3d.logger.info("*******Coil resistance increased by {:.2f}%".format(resistance_increase))
+m3d.logger.info("*******Ohmic loss in coil AFTER temperature feedback =  {:.2f}W".format(EM_loss_new/1000))
```

Comment:
```suggestion
m3d.logger.info("*******Ohmic loss in coil AFTER temperature feedback =  {:.2f}W".format(em_loss_new/1000))
```
remember to change the variable where is used in the code, otherwise it fails :)